recipes/apriltag/all/conanfile.py

```diff
@@ -58,16 +58,30 @@ def source(self):
 
     def generate(self):
         tc = CMakeToolchain(self)
+        tc.cache_variables["BUILD_EXAMPLES"] = False
         if Version(self.version) >= "3.1.4":
             tc.variables["BUILD_PYTHON_WRAPPER"] = False
         tc.variables["CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS"] = True
+        if self.settings.os == "Windows":
+            tc.preprocessor_definitions["NOMINMAX"] = ""
         tc.generate()
         if is_msvc(self):
             deps = CMakeDeps(self)
             deps.generate()
 
-    def build(self):
+    def _patch_sources(self):
         apply_conandata_patches(self)
+        # Fix DLL installation
+        replace_in_file(self, os.path.join(self.source_folder, "CMakeLists.txt"),
```

Comment:
Created a PR to the upstream fixing it: https://github.com/AprilRobotics/apriltag/pull/300

Reply:
Thanks a lot!!!